.github/renovate.json5

```diff
@@ -8,6 +8,8 @@
   ignorePaths: [
     '**/tests/**',
   ],
+  // See rust-lang/cargo#13546 and openssl/openssl#23376 for the exclusion
+  ignoreDeps: ['openssl', 'openssl-src', 'openssl-sys'],
```

Comment:
For reference: <https://docs.renovatebot.com/configuration-options/#ignoredeps>.

I am fine merging this as-is. 

Do you think we should remove the exact requirement from `openssl` crate?

Reply:
> Do you think we should remove the exact requirement from openssl crate?

Probably worth keeping to avoid accidental upgrades (e.g. changing another dep causes it to be upgraded) as the manifest has a bit more visibility